pkg/background/background.go

```diff
@@ -0,0 +1,27 @@
+package background
+
+import (
+	"os"
+
+	chart "github.com/werf/common-go/pkg/lock"
+	"github.com/werf/lockgate"
+)
+
+func IsBackgroundModeEnabled() bool {
+	return os.Getenv("_WERF_BACKGROUND_MODE_ENABLED") == "1"
+}
+
+// TryLock tries background host-lock or panics if error.
+func TryLock() bool {
```

Comment:
Why do you use panic instead of handling the error?

Reply:
We use this function in background mode only and we are forced to use it BEFORE logger initialization. If we return the error from `TryLock` how should we handle that error?